doc/factory/troubleshooting.html

```diff
@@ -269,7 +269,8 @@ <h2 id="colorcoded">Security Overview</h2>
             ><font color="chocolate"
```

Comment:
I'm not sure I'd add this file to the same PR
It has nothing to do with it. Strange that pre-commit even touched it